client/src/api/routes/applications.ts

```diff
@@ -43,14 +43,15 @@ export const useGetApplicationFromCycle = (cycleId: string, applicationId: strin
  * @returns {UseQueryResult<ApplicationObject>} the newly created application
  * @throws if fails
  */
-export const useCreateApplication = (company: string, position: string, location: string, jobPostUrl: string, description: string, applyDate: Date): UseQueryResult<ApplicationObject> => {
+export const useCreateApplication = (company: string, position: string, location: string, jobPostUrl: string, description: string, salary: number, applyDate: Date): UseQueryResult<ApplicationObject> => {
```

Comment:
salary should be removed from the useCreateApplication stuff, since the backend doesn't use it

Reply:
The forms should also not have it, they won't know salary until they get an offer anyway, most of the time.